base_layer/p2p/tests/saf/mod.rs

```diff
@@ -58,6 +58,7 @@ fn create_peer_storage(tmpdir: &TempDir, database_name: &str, peers: Vec<Peer>)
         .unwrap();
 
     let peer_database = datastore.get_handle(database_name).unwrap();
+    let peer_database = LMDBWrapper::new(Arc::new(peer_database));
```

Comment:
This code snippet gets called quite a lot, perhaps we can add it to a common file?

Reply:
Yes, I was thinking something like this too.. but it's mostly in tests though.